docs/docs/story-format.md

```diff
@@ -4,28 +4,25 @@
 
 Any ModuleScript with a `.storybook` extension will be picked up as a Storybook.
 
-:::warning
-Storybooks that do not have a `storyRoots` array will not be shown in the Flipbook UI.
-:::
-
 The properties that can be used in the module are as follows:
 
-| **Property** | **Type**             | **Description**                                                                                                                       |
-| ------------ | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| `storyRoots` | `{ Instance }`       | Locations that the Storybook manages. Each instance will have its descendants searched for Story modules.                             |
-| `name`       | `string?`            | An optional name for the Storybook. Defaults to the module name with the extension removed. i.e. `Sample.storybook` becomes `Sample`. |
-| `packages`   | `{ [string]: any }?` | An optional dictionary used for supplying the Storybook with the packages to use when rendering its Stories.                          |
+| **Property**    | **Type**                                        | **Description**                                                                                                                       |
+| --------------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `storyRoots`    | `{ Instance }`                                  | Locations that the Storybook manages. Each instance will have its descendants searched for Story modules.                             |
+| `name`          | `string?`                                       | An optional name for the Storybook. Defaults to the module name with the extension removed. i.e. `Sample.storybook` becomes `Sample`. |
+| `packages`      | `{ [string]: any }?`                            | An optional dictionary used for supplying the Storybook with the packages to use when rendering its Stories.                          |
+| `mapDefinition` | `((story: any) -> any)?`                        |                                                                                                                                       |
+| `mapStory`      | `((story: any) -> (props: StoryProps) -> any)?` |                                                                                                                                       |
```

Comment:
Either add support for mapStory and mapDefinition to all renderers or mark them as Roblox internal only and remove mention from the docs